builder/ConfTargets.cmake

```diff
@@ -121,5 +123,7 @@ if (DEFINED MFX_CONFIG_FILE)
     include(${MFX_CONFIG_FILE})
 endif()
 
+add_definitions( " -DMFX_PRODUCT_NAME='\"${MFX_PRODUCT_NAME}\"'" )
```

Comment:
No. You don't need this. Instead add the following in the mfxconfig.h.in:
`#define MFX_PRODUCT_NAME "@MFX_PRODUCT_NAME@"`